doc/release-process.md

```diff
@@ -87,7 +87,7 @@ Ensure gitian-builder is up-to-date:
     pushd ./gitian-builder
     mkdir -p inputs
     wget -P inputs https://bitcoincore.org/cfields/osslsigncode-Backports-to-1.7.1.patch
-    wget -P inputs http://downloads.sourceforge.net/project/osslsigncode/osslsigncode/osslsigncode-1.7.1.tar.gz
+    wget -P inputs https://downloads.sourceforge.net/project/osslsigncode/osslsigncode/osslsigncode-1.7.1.tar.gz
```

Comment:
If they are not checked, https isn't going to help either.

Reply:
Yes, ideally we should have both: checksumming to make sure we get the data we except and TLS to avoid leaking our request to eavesdroppers (beyond leaking the hostname).